### PR TITLE
add --hostname option

### PR DESCRIPTION
**Pull request by rmaksimov**
@mohemiv noticed that ruler uses predefined value "RULER" while using NTLM authentication
so i have added the --hostname option to specify any value as a host name

### CODE REVIEW

File: ruler.go [hunk 1]
```diff
@@ -1220,6 +1212,11 @@ A tool by @_staaldraad from @sensepost to abuse Exchange Services.`
 			Name:  "rpc",
 			Usage: "Force RPC/HTTP rather than MAPI/HTTP",
 		},
+		cli.StringFlag{
+			Name:  "hostname,n",
+			Value: "",
```
Comment (staaldraad) [inline ruler.go:1217]: Could we leave the default value as RULER. 

It is one of the IOCs we published for the tool and although easily modifiable, it is worth having as a "script kiddie" or basic detection effort built into the tool. 

https://github.com/sensepost/notruler/blob/master/iocs.md#local-host-domain-controllerexchange-server

Comment (rmaksimov) [reply ruler.go]: Sure, I forgot to set it as the default value

**Comment by rmaksimov** [comment]
It should be noted that the current PR solves only part of a problem (related to your fork of go-ntlm). I'm going to fix this in the near future, but it should be applied in both projects at the same time.
Until then, you can probably leave this PR open

**Comment by staaldraad** [comment]
Awesome, thanks for the update. Yeah I had a look at updating it in the dependency as well, however this looks like it correctly overwrites the value hard-coded into the defaults. I'm happy to leave it as is, and pull this into Ruler.

**Pull request merged**